generators/app/index.js

```diff
@@ -259,7 +342,7 @@ module.exports = class extends Generator {
   }
 
   updateRegistry() {
-    const registryFile = '../content-build/src/applications/registry.json';
+    const registryFile = 'src/applications/registry.json';
```

Comment:
This change is preventing the `registry.json` file from being updated locally in the `content-build` repo. `../content-build` should still be prepended here

Reply:
> This change is preventing the `registry.json` file from being updated locally in the `content-build` repo. `../content-build` should still be prepended here

This change must've snuck in somehow as it was not supposed to be there, will remove.